docs/debuginfod.md

```diff
@@ -0,0 +1,37 @@
+# debuginfod client
+
+In order to symbolize ingested profiles, Parca needs to have debug information
+for the binaries _(which binaries?)_ that are being profiled. Debug information
+can typically comprises ELF object files, DWARF debug data, and source code
+required for symbolization. However, sometimes, production codebases use
+compiler optimisations which strip the debug information from the binaries.
+Thankfully, there are public vendor distributed servers which make distro
+specific debug info files avaialble to the community.
+
+debuginfod is an eifutils file server that serves debug information to clients
```

Comment:
I don't think it's important in this context which group of binaries debuginfod belongs to.